CrossPlatformDesktopProject/CrossPlatformDesktopProject/Link/Equipables/InventoryManager.cs

```diff
@@ -56,6 +58,18 @@ public void UseBow(ButtonKind envokedWith)
             currentItem = bow;
         }
 
+        public void UseBow(ButtonKind envokedWith)
```

Comment:
GitHub doubled the `UseBow` method for some reason when I committed it to my branch. I will remove this, and that should fix the only error that I found.